examples/js/controls/OrbitControls.js

```diff
@@ -1136,7 +1136,15 @@ THREE.OrbitControls = function ( object, domElement ) {
 	scope.domElement.addEventListener( 'touchend', onTouchEnd, false );
 	scope.domElement.addEventListener( 'touchmove', onTouchMove, false );
 
-	window.addEventListener( 'keydown', onKeyDown, false );
+	scope.domElement.addEventListener( 'keydown', onKeyDown, false );
+
+	// make sure element can receive keys.
+
+	if ( scope.domElement.tabIndex === -1 ) {
```

Comment:
Formatting:

    if ( scope.domElement.tabIndex === - 1 ) {

But instead, I would avoid [guessing](https://github.com/mrdoob/three.js/pull/16676/files#r298252234) what the user's intention is and remove this test.

Reply:
Done #17826.

@EliasHasle Please consider to setup your IDE/editor in a way so this style issue are automatically detected.

Reply:
Guys. The white space is not the issue. Leave `tabIndex` to the application.

Reply:
Have you actually tested the example without this code?

Reply:
@Mugen87 I recall, now, that this came up before in #4327. See https://github.com/mrdoob/three.js/commit/ae9934776f383547ef136c13ed0b289a0d2abbb4.